docker/docker-compose.prod.yml

```diff
@@ -15,26 +15,13 @@ x-server-build: &server-common
 services:
   immich-server:
     container_name: immich_server
-    command: ['start.sh', 'immich']
     <<: *server-common
```

Comment:
idem

Reply:
Honestly do we even use this anymore? I never use it anymore since `make e2e` exists, which does the same thing but also can run tests, etc.